test/test_images/deadstart/README.md

```diff
@@ -0,0 +1,4 @@
+# Deadstart test image
```

Comment:
I added an image here to help with this

https://github.com/knative/serving/pull/14875

Reply:
Going to convert the test to use this image.